tt_metal/jit_build/genfiles.cpp

```diff
@@ -19,14 +19,44 @@ using namespace std;
 
 namespace tt::tt_metal {
 
-static void gen_trisc_cpp(const string& src_name, const string& dst_name, vector<string>& prolog)
+static void gen_risc_cpp(const string& src_name, const string& dst_name, vector<string>& prolog)
```

Comment:
all looks good except this-I don't like "risc" as a name as it is a coincidence that we use risc processors.  rename to "gen_kernel_cpp"?

Reply:
Yeah wasn't sure what name would be good here, updated it to "gen_kernel_cpp"